en/core-libraries/time.rst

```diff
@@ -140,6 +140,23 @@ dates. CakePHP makes this a snap::
     // Outputs '2014-04-20 22:10'
     $now->i18nFormat('YYYY-MM-dd HH:mm:ss');
 
+As of 3.1.0 you can format dates with non-gregorian calendars::
+
+    // Outputs 'Thursday, Dey 24, 1388 at 1:59:28 PM GMT'
```

Comment:
Oh poop, I copied the example from the tests, and forgot that 2014-10-31 is not the same as it was there. I'll update that snippet.